config/responsecache.php

```diff
@@ -34,4 +34,9 @@
      * configured in app/config/cache.php
      */
     'cache_store' => env('RESPONSE_CACHE_DRIVER', 'file'),
+
+    /*
+     * If selected cache driver supports tagging you can define it here.
```

Comment:
I think we should add some extra info here, to help explain the benefits of this features to newcomers. It should be an answer to "Why would I use tags?"

Reply:
tags would be useful if someone needs to invalidate cache on a section / module within the site also the tags could be added from controllers for dynamic purposes